lib/Runtime/Debug/TTEventLog.cpp

```diff
@@ -2818,6 +2819,8 @@ namespace TTD
         TTDAssert(wcscmp(_u("x64"), archString.Contents) == 0, "Mismatch in arch between record and replay!!!");
 #elif defined(_M_ARM)
         TTDAssert(wcscmp(_u("arm64"), archString.Contents) == 0, "Mismatch in arch between record and replay!!!");
+#elif defined(_M_ARM64)
+        TTDAssert(wcscmp(_u("arm64"), archString.Contents) == 0, "Mismatch in arch between record and replay!!!");
```

Comment:
Curious why _M_ARM and _M_ARM64 both test for "arm64"

Reply:
The M_ARM path may be wrong but I don't have testing set up for ARM or a device to check it on, so was nervous about changing it - it's a very old line of code and before microsoft stopped supporting CC it used to pass arm tests on windows, implying perhaps that both sides of the condition were being set incorrectly.

Reply:
Can you add a comment to look at it later?

Reply:
Will do (let me know when you've finished reviewing and I'll address all points together if that's ok)